plugins/wporg-5ftf/views/form-my-pledges.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace WordPressDotOrg\FiveForTheFuture\View;
+use WP_User, WP_Post;
+
+/**
+ * @var WP_User $user
+ * @var array   $contributor_posts
+ * @var WP_Post $contributor_post
+ * @var string  $success_message
+ * @var string  $pledge_url
+ */
+
+?>
+
+<?php if ( is_user_logged_in() ) : ?>
+
+	<?php echo get_avatar( $user->ID, 96, 'blank', "{$user->login}'s avatar" ); ?>
+
+	<p>Pledged 10 hours a week across two organizations</p> <?php // todo pull from profiles. ?>
+
+	<?php if ( $success_message ) : ?>
+		<div class="notice notice-success notice-alt">
+			<p>
+				<?php echo esc_html( $success_message ); ?>
+			</p>
+		</div>
+	<?php endif; ?>
+
+	<?php if ( $contributor_posts ) : ?>
+		<?php foreach ( $contributor_posts as $contributor_post ) : ?>
+			<?php $pledge = get_post( $contributor_post->post_parent ); ?>
+
+			<?php echo get_the_post_thumbnail( $pledge->ID, 50 ); ?>
```

Comment:
This size doesn't work (it needs to be a name or an array) - but I added a custom size in #43, so you can use `pledge-logo`.

Reply:
Haven't started looking at any of the design stuff yet, but I'll update the size.